azure-proto-network/NetworkInterfaceContainer.cs

```diff
@@ -69,9 +69,9 @@ public async override Task<ArmOperation<NetworkInterface>> StartCreateAsync(stri
         /// </summary>
```

Comment:
I would expect each of the [Resource]Operations classes to have the listavailablelocations/async methods on them inside azure_proto_network just like azure_proto_compute except for SubnetOperations because it is a subresource not a trackedresource